### PR TITLE
DROOLS-4957: Adapt ScenarioSimulation Wrapper for Kogito project

### PR DESCRIPTION
**Pull request by yesamer**
@danielezonca @gitgabrio @jomarko 

@jomarko We decided to split https://github.com/kiegroup/drools-wb/pull/1281 in multiple sub-tickets, in order to manage better item DROOLS-3979.
In this PR, I added all required changes on `drools-wb-scenario-simulation-editor-client` and `drools-wb-scenario-simulation-editor-businesscentral-client`. Minimal changes, mostly on `Wrapper` related classes.

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java [hunk 1]
```diff
@@ -360,12 +361,8 @@ protected void save(final String commitMessage) {
...
     @Override
     protected Command getBeforeSaveAndRenameCommand() {
-        return this::synchronizeColumnsDimension;
-    }
-
-    protected void synchronizeColumnsDimension() {
-        scenarioSimulationEditorPresenter.getContext().getScenarioGridPanelByGridWidget(GridWidget.SIMULATION).synchronizeFactMappingsWidths();
-        scenarioSimulationEditorPresenter.getContext().getScenarioGridPanelByGridWidget(GridWidget.BACKGROUND).synchronizeFactMappingsWidths();
+        return () -> synchronizeColumnsDimension(scenarioSimulationEditorPresenter.getContext().getScenarioGridPanelByGridWidget(GridWidget.SIMULATION),
+                                           scenarioSimulationEditorPresenter.getContext().getScenarioGridPanelByGridWidget(GridWidget.BACKGROUND));
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java]: Formatting

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/popup/FileUploadPopup.java [hunk 2]
```diff
@@ -38,12 +38,16 @@ void show(final List<String> acceptedExtension,
                   final Command okCommand);
 
         String getFileContents();
+
+        String getFileName();
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/popup/FileUploadPopup.java]: I think it is better not to add the method if we do not need it currently. Can happen in final won't be needed at all but we will forget to remove it from API.

**Comment by jomarko** [comment]
Not able to build locally, maven says about every singe kiegroup jar that can not be downloaded.

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/popup/FileUploadPopup.java (hunk 2 above)
Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/popup/FileUploadPopup.java]: @jomarko I was wrong, it is used is `ScenarioSimulationEditorKogitoTestingScreen` (present in branch 3879 only). Please ignore my previous comment.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java (hunk 1 above)
Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/main/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/editor/ScenarioSimulationEditorBusinessCentralWrapper.java]: @jomarko Done, thank you

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Pull request merged**